gtk4/src/auto/functions.rs

```diff
@@ -148,10 +149,21 @@ pub fn disable_setlocale() {
     }
 }
 
-//#[doc(alias = "gtk_distribute_natural_allocation")]
-//pub fn distribute_natural_allocation(extra_space: i32, n_requested_sizes: u32, sizes: /*Ignored*/&mut RequestedSize) -> i32 {
-//    unsafe { TODO: call ffi:gtk_distribute_natural_allocation() }
-//}
+#[doc(alias = "gtk_distribute_natural_allocation")]
+pub fn distribute_natural_allocation(
+    extra_space: i32,
+    n_requested_sizes: u32,
+    sizes: &mut RequestedSize,
```

Comment:
This is wrong. It's supposed to be `fn distribute_natural_allocation(extra_space: i32, sizes: &[RequestSize]) -> i32`. And unclear if the sizes should be mutable or not, are they going to be changed in there?

Also the C function is missing the array annotations :)

Reply:
I fixed the annotations upstream and updated the function here. Sorry, I should have been a bit more careful here :)

Reply:
> And unclear if the sizes should be mutable or not, are they going to be changed in there?

Yes the minimum size is computed by this function